.github/workflows/weekly.yml

```diff
@@ -1,14 +1,15 @@
-name: Weekly Beta
+name: Weekly CI
 
 on:
+  push: # TODO: remove before merging
   schedule:
     - cron: "0 0 * * 1" # every monday at 00:00
 
 env:
   RUSTFLAGS: -Dwarnings
 
 jobs:
-  test:
+  weekly-clippy-beta:
```

Comment:
huh, is it this job name which must be unique across all workflows for the clippy action to get this right?